src/account/AccountInfo.js

```diff
@@ -301,9 +307,17 @@ export default class AccountInfo {
                 info.ledgerId != null
                     ? LedgerId.fromBytes(info.ledgerId)
                     : null,
-
+            /*
+             * @deprecated - no longer supported
```

Comment:
I don't think these need to be marked, heh.

Reply:
ok, updated:)